SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs

```diff
@@ -19,10 +19,12 @@ public partial class Crashes : IAppCenterService
         /// </summary>
         public bool InstanceEnabled { get; set; }
 
+        private readonly ErrorReport LastSessionErrorReport = null;
```

Comment:
```suggestion
        private readonly ErrorReport _lastSessionErrorReport = null;
```

Reply:
Yeah definitely, Resharper should yell to rename it also.

Reply:
Thing is, it should not be readonly when we'll do the work for real, should actually use a property since we access it via `Instance.`, if you use a property you can keep PascalCase.